batch/batch/front_end/templates/openapi.yaml

```diff
@@ -0,0 +1,1449 @@
+openapi: 3.1.0
+info:
+  version: 0.2.133
```

Comment:
This could be a bit of a pain to keep updated. It's also not technically correct, since changes to the service api are deployed continuously, not when we release a new version. Does this actually matter much?

Actually, I think we could template this. `front_end.py` already imports `version`, which is the full hail version, including the commit hash. That would be most accurate. But if you think we'd rather have just the release version, you could also import `pip_version` from `hailtop`.

Reply:
Hmm, good catch. This is not really necessary, at least for the purposes of this POC. I think if we were publishing multiple versions of our API spec we might want to be able to switch between or identify which one we're looking at, but that's definitely not a _now_ requirement. For now I'll just remove this field since it's already out of date.

Reply:
ah, it's a required field. I'll just template in the `version`